baseline/OSPS-AC.yaml

```
@@ -183,6 +193,10 @@ controls:
           - 347-507
           - 263-284
           - 123-124
+        - reference-id: SLSA
+          identifiers:
+          - Producer - Choose an appropriate build platform
+          - Build platform - Isolation strength - Isolated
```

```suggestion
      - reference-id: SLSA
        identifiers:
          - Producer - Choose an appropriate build platform
          - Build platform - Isolation strength - Isolated
```

I lied. This fixes it. I should have tried locally first :-(